collect_app/src/main/java/org/odk/collect/android/projects/AddProjectDialog.kt

```diff
@@ -0,0 +1,63 @@
+package org.odk.collect.android.projects
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.Toolbar
+import org.odk.collect.android.R
+import org.odk.collect.android.databinding.AddProjectDialogLayoutBinding
+import org.odk.collect.android.injection.DaggerUtils
+import org.odk.collect.material.MaterialFullScreenDialogFragment
+import javax.inject.Inject
+
+class AddProjectDialog : MaterialFullScreenDialogFragment() {
```

Comment:
I think we need tests to check that clicking "Cancel" or pressing back dismiss the dialog. As far as I can tell we should be able to do that with the `FragmentScenario` tests (`AudioRecordingFormErrorDialogFragmentTest` is a recent example).

Reply:
Ok, I added those tests.